types/currency_test.go

```diff
@@ -222,6 +222,10 @@ func TestCurrencyEncoding(t *testing.T) {
 // TestNegativeCurrencyMulRat checks that negative numbers are rejected when
 // calling MulRat on the currency type.
 func TestNegativeCurrencyMulRat(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+
```

Comment:
I tried running `go test -short` with and without `debug`, and the time difference seemed pretty negligible, even on my laptop. No package took more than 100ms longer to complete with `debug`. What's the slowdown like on your laptop?